core/app/models/workarea/order.rb

```diff
@@ -253,18 +252,23 @@ def add_item(attributes)
       save
     end
 
-    # Updates an items attributes
+    # Update an item's attributes. Increases quantity by the provided
```

Comment:
This method didn't change in its feature set, but the docs were just lacking a bit.

Reply:
This comment is misleading. Quantity is only increased by the value passed, rather than set to that value, if you pass an id of an item that doesn't match the item found by `find_existing`, as its essentially merging items at that point. Under normal conditions `update_item` sets the value of the item matching the id passed to the values passed.

Reply:
I'll clarify that part, thanks.